gr-qtgui/python/qtgui/compass.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+
+import numpy
+from gnuradio import gr;
+import pmt
+import time
+
+#from PyQt5.QtWidgets import QApplication, QWidget,  QVBoxLayout, QHBoxLayout, QGridLayout,  QSpinBox,  QSlider
+#from PyQt5.QtCore import QSize, pyqtSignal,  QPoint,  pyqtSlot,  pyqtProperty,  Qt
+#from PyQt5.QtGui import QPainter, QPalette,  QFont,  QFontMetricsF,  QPen,  QPolygon
+
+# First Qt and 2nd Qt are different.  You'll get errors if they're both not available, hence the import-as to avoid name collisions
```

Comment:
This line is too long (72chars +) Please make it a 2 liner comment, and also please run pylint on all your python files here and it'll detect this and other errors

Reply:
Sorry, I confused the suggested git commit line length (72) here.

GR's coding guide lines is maximum 100 chars, ideally 80 if sensible.